pipelines/whale/loader/whale_loader.py

```diff
@@ -216,9 +216,7 @@ def _update_table_metadata(sections, record):
 
 
 def _update_index_metadata(sections, record):
-    if not sections[INDEX_SECTION]:
-        sections[INDEX_SECTION] = INDEX_DELIMITER + "\n"
-
+    sections[INDEX_SECTION] = INDEX_DELIMITER + "\n"
```

Comment:
Ah this almost works, except if there are multiple indices, only the last one shows up :/

Reply:
🤦 Will take another look today.